### PR TITLE
[Linux] [SOS.NETCore] Quick fix for Path.GetFileName

### PR DESCRIPTION
**Pull request by Dmitri-Botcharnikov**
This patch is a quick fix for issue #7184 that enables `SOS.NETCore` on Linux to read debugging information generated on Windows.
@mikem8361 @seanshpark @chunseoklee @lucenticus @kvochko @ayuckhulk

### CODE REVIEW

File: src/ToolBox/SOS/NETCore/SymbolReader.cs [hunk 1]
```diff
@@ -126,6 +126,27 @@ public override void Write(byte[] buffer, int offset, int count)
...
+        /// Quick fix for Path.GetFileName which incorrectly handles Windows-style paths on Linux
+        /// </summary>
+        /// <param name="pathName"> File path to be processed </param>
+        /// <returns>Last component of path</returns>
+        private static string GetFileName(string pathName)
+        {
+            int pos = -1;
+            for (int i = pathName.Length - 1; i >= 0; --i)
+            {
+                if (pathName[i] == '/' || pathName[i] == '\\')
```
Comment (mikem8361) [inline src/ToolBox/SOS/NETCore/SymbolReader.cs]: You could use pathName.IndexOfAny(new char[] { '/', '\\'}) instead of this code.

Comment (mikem8361) [inline src/ToolBox/SOS/NETCore/SymbolReader.cs]: You could use pathName.IndexOfAny(new char[] { '/', '\\'}) instead of this code.

Comment (Dmitri-Botcharnikov) [inline src/ToolBox/SOS/NETCore/SymbolReader.cs]: @mikem8361 thank you for pointing this.

**Comment by mikem8361** [comment]
LGTM

**Pull request merged**